### PR TITLE
[WIP] Install KEDA for testing

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

**Comment by pierDipi** [comment]
@aavarghese with the default values we're slower (higher e2e latency) in our Sacura test: https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative-sandbox_eventing-kafka-broker/2796/reconciler-tests_eventing-kafka-broker_main/1594762319668187136

**Comment by aavarghese** [comment]
@pierDipi Do you see higher latency errors in the failing tests?
I found a few other errors in there:

```
              {
                "lastTransitionTime": "2022-11-22T16:30:29Z",
                "message": "failed to set up autoscaler: SASL type value \"\" is not supported",
                "reason": "AutoscalerFailed",
                "status": "False",
                "type": "ConsumerGroup"
              },
```
and
```
{
            "type": "ConsumerGroup",
            "status": "False",
            "lastTransitionTime": "2022-11-22T16:08:15Z",
            "reason": "AutoscalerFailed",
            "message": "failed to set up autoscaler: failed to update scaled object eventing-e2e19/so-475d6208-d85e-474c-a377-85e1e57206ce: scaledobjects.keda.sh \"so-475d6208-d85e-474c-a377-85e1e57206ce\" is invalid: metadata.resourceVersion: Invalid value: 0x0: must be specified for an update"
           },
```

**Comment by pierDipi** [comment]
> @pierDipi Do you see higher latency errors in the failing tests? I found a few other errors in there:
 
No, however, I think we should allow configuring default values for the autoscaler via a ConfigMap because scale to 0 by default might have a big latency impact

File: hack/update-keda.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# quick check - we need exactly one argument which denotes the Strimzi version we want to update to
+if [[ "$#" -ne 1 ]]; then
+  echo "Usage: $0 <strimzi_version>"
+  exit 1
```
Comment (Cali0707) [inline hack/update-keda.sh:20]:
```suggestion
# quick check - we need exactly one argument which denotes the KEDA version we want to update to
if [[ "$#" -ne 1 ]]; then
  echo "Usage: $0 <keda_version>"
  exit 1
```
Shouldn't these be keda versions?

File: test/keda/keda_setup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Knative Authors
```
Comment (Cali0707) [inline test/keda/keda_setup.sh:3]:
```suggestion
# Copyright 2023 The Knative Authors
```

File: hack/update-keda.sh (hunk 1 above)
Comment (pierDipi) [reply hack/update-keda.sh:20]: Yes

**Comment by Cali0707** [comment]
@pierDipi can we close this PR now that #3285 is merged?

**Pull request closed without merging**